src/content/cloud/registry.mdx

```diff
@@ -50,3 +50,25 @@ Any image pushed into the Okteto Registry is private. You'll need to authenticat
 Namespaces in Okteto Cloud are automatically allowed to pull images that belong to their namespace automatically. If your application uses images from the Okteto Registry, it'll be able to pull container images without any extra configuration.
 
 To pull an image from a different namespace, you'll need to create and configure the required `imagePullSecrets` as outlined [here](/docs/reference/faqs/#how-to-use-private-images).
+
+## Push helm chart into the Okteto Registry
+
+Just as we can push images to the Okteto Registry, you also have the option to push Helm charts. To do this, the first step is to log in to the registry where we want to push the chart:
+
+```
+helm registry login registry.cloud.okteto.net -u my-username -p my-password
+```
+
+Once you are logged against the Okteto Registry you can push the packaged chart to the Okteto Registry by using the command below:
+
+```
+helm push ./movies-api-0.1.0.tgz oci://registry.cloud.okteto.net/my-namespace
+```
+
+Once the packaged chart has been pushed to the Okteto Registry, you can pull it by using the command below:
+
+```
+helm pull oci://registry.cloud.okteto.net/my-namespace/movies-api --version 0.1.0
+```
+
+> Note: This example is using the registry for Cloud. You need adapt the registry domain and namespace name to your case using the notation `oci://<registry-domain>/<okteto-namespace>`.
```

Comment:
how about creating a post as a `How Do I` in our community?

Reply:
> Based on the above, I would remove this note.
> 
> Also, the main use case for this feature is to push a chart to the global registry (`okteto` namespace), and pull it from any namespace. This is useful if you have a single repo with all your charts. On each merge, you push these charts to the `okteto` namespace (requires admin token). Then, any developer can pull the charts.
> 
> @rberrelleza @AdrianPedriza where would you have the use case explained?

We don't talk about global registry. Canwe add a section on how to use the global registry here? 

For the scenario you mention, I think a blog post or how to in community is a better place.

Reply:
should I then add a section on using the global registry to push charts? @rberrelleza @pchico83  isn't it a specific use case that should go as a blog post or something similar?

Reply:
Since the issue already has several approvals, how about merging the PR with the documentation so far and creating a new issue about how to document the use of the global registry as a chart store? so we can close this sprint tasks and continue discussing the best place to document this use case. @rberrelleza @pchico83

Reply:
https://github.com/okteto/okteto/issues/3160